egs/ljspeech/TTS/vits/tokenizer.py

```diff
@@ -63,30 +76,44 @@ def texts_to_token_ids(self, texts: List[str], intersperse_blank: bool = True):
             # Text normalization
             text = tacotron_cleaner.cleaners.custom_english_cleaners(text)
             # Convert to phonemes
-            tokens = self.g2p(text)
+            tokens_list = phonemize_espeak(text, "en-us")
```

Comment:
Please pass `en-us` as an argument to this function.
You can use
```python3
lang: str = 'en-us`
```
as the last argument for this function

Reply:
thanks a lot!